server/sonar-server/src/main/java/org/sonar/server/platform/ServerComponents.java

```diff
@@ -680,7 +679,6 @@ void startLevel4Components(ComponentContainer pico) {
     pico.addSingleton(IssueService.class);
     pico.addSingleton(IssueActionsWriter.class);
     pico.addSingleton(IssueQueryService.class);
-    pico.addSingleton(IssueNotifications.class);
```

Comment:
the notifications are used only in the context of computation stack ? Otherwise, I don't understand why you remove this component